build/run_pre_reqs.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+#(C) Copyright [2020] Hewlett Packard Enterprise Development LP
+#
+#Licensed under the Apache License, Version 2.0 (the "License"); you may
+#not use this file except in compliance with the License. You may obtain
+#a copy of the License at
+#
+#    http:#www.apache.org/licenses/LICENSE-2.0
+#
+#Unless required by applicable law or agreed to in writing, software
+#distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#License for the specific language governing permissions and limitations
+# under the License.
+
+# perform pre-requisites required for creating docker image
+pre_reqs()
+{
+	if [[ -z ${ODIMRA_GROUP_ID} ]] || [[ -z ${ODIMRA_USER_ID} ]]; then
+		echo "[$(date)] -- ERROR -- ODIMRA_GROUP_ID or ODIMRA_USER_ID is not set, exiting"
+		exit 1
+	fi
+
+	if [[ -n "$(getent passwd odimra 2>&1)" ]]; then
+		echo "[$(date)] -- INFO  -- user odimra exists"
+		sudo userdel odimra
+	fi
+	if [[ -n "$(getent group odimra 2>&1)" ]]; then
+		echo "[$(date)] -- INFO  -- group odimra exists"
+		sudo groupdel odimra
+	fi
+	sudo groupadd -g ${ETCD_GROUP_ID} -r odimra
```

Comment:
Please change ETCD_GROUP_ID to ODIMRA_GROUP_ID, and same in next line for ETCD_USER_ID.

Reply:
Done. Changed it to ODIMRA.